docker/test/stateless/run.sh

```diff
@@ -246,16 +246,19 @@ clickhouse-client -q "system flush logs" ||:
 stop_logs_replication
 
 # Try to get logs while server is running
-successfuly_saved=0
+failed_to_save_logs=0
 for table in query_log zookeeper_log trace_log transactions_info_log metric_log
 do
-    clickhouse-client -q "select * from system.$table format TSVWithNamesAndTypes" | zstd --threads=0 > /test_output/$table.tsv.zst
-    successfuly_saved=$?
+    err=$( { clickhouse-client -q "select * from system.$table format TSVWithNamesAndTypes" | zstd --threads=0 > /test_output/$table.tsv.zst; } 2>&1 )
+    echo $err
```

Comment:
If we want to keep the previous behavior it should be redirected to stderr (and the same below)

Reply:
I think it does not matter where it is redirected, it matters that we see it in `run.log`.

Reply:
> If we want just to check exit code of clickhouse as well as zstd, maybe just use  set -o pipefail? At least for this part of the script.

Is it ok if I set it in a separate follow-up PR? I want this fix for run.sh asap to debug another issue...

Reply:
Sure, maybe it is not needed at all, if this works